kickoff/templates/mobile_details.json

```diff
@@ -5,6 +5,12 @@
   "version": "{{ versions.stable }}",
   "ios_beta_version": "{{ versions.ios_beta_version }}",
   "ios_version": "{{ versions.ios_version }}",
+  "locales": {
```

Comment:
I would rather put them in https://github.com/mozilla-releng/ship-it/blob/master/kickoff/config.py
as we might use this list in other places